wire-ios-request-strategy/Sources/Request Strategies/Conversation/Actions/CreateGroupConversationActionHandler.swift

```diff
@@ -140,6 +141,44 @@ final class CreateGroupConversationActionHandler: ActionHandler<CreateGroupConve
         case (403, "access-denied"):
             action.fail(with: .accessDenied)
 
+        case (409, _):
+            let errorInfo = response.errorInfo
+            guard
+                let payload = Payload.ErrorResponse(response),
+                let nonFederatingDomains = payload.non_federating_backends
+            else {
+                return action.fail(with: .unknown(
+                    code: errorInfo.status,
+                    label: errorInfo.label,
+                    message: errorInfo.message
```

Comment:
I would return a `.processingError` here, since we know what the error should be but we not able to parse it correctly.

Reply:
Done